packages/webdriverio/src/middlewares.js

```diff
@@ -51,6 +51,48 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
             }).apply(this)
         }
 
+        /**
+         * apply workarounds for unclickable elements if:
+         *  - elementId can be fetched
+         *  - command to click element would return "not clickable" error
+         */
+        if (this.elementId && commandName.match(/click/)) {
```

Comment:
I am not sure if we need to do a regex check here. I think it is easier to just do:

```js
if (this.elementId && commandName === 'click') {
```

Reply:
:100: